test/e2e/network_test.go

```diff
@@ -146,12 +147,19 @@ var _ = Describe("Podman network", func() {
 	It("podman network ID test", func() {
 		net := "networkIDTest"
 		// the network id should be the sha256 hash of the network name
-		netID := "6073aefe03cdf8f29be5b23ea9795c431868a3a22066a6290b187691614fee84"
+		netID := "6073aefe03cdf8f29be5b23ea9795431868a3a22066a6290b187691614fee84"
 		session := podmanTest.Podman([]string{"network", "create", net})
 		session.WaitWithDefaultTimeout()
-		defer podmanTest.removeCNINetwork(net)
+		defer podmanTest.removeNetwork(net)
 		Expect(session).Should(Exit(0))
 
+		if podmanTest.NetworkBackend == Netavark {
+			// netavark uses a different algo for determining the id and it is not repeatable
+			getid := podmanTest.Podman([]string{"network", "inspect", net, "--format", "\"{{.ID}}\""})
```

Comment:
```suggestion
			getid := podmanTest.Podman([]string{"network", "inspect", net, "--format", "{{.ID}}"})
```
do not add quotes when your remove them below anyway

Reply:
fixed